ModuleNotFoundError in react-addons-update

Using 
```
alias: {
            'react': 'preact-compat',
            'react-dom': 'preact-compat'
        }
```
in existing react app, I get
```
ModuleNotFoundError: Module not found: Error: Cannot resolve module 'preact-compat/lib/update' in %some-long-path-here%/node_modules/react-addons-update
```
I'm using react@15.1.0 and react-addons-update@15.1.0.
Do you happen to know whether there's a fix to this issue?

--- a/lib/update.js
+++ b/lib/update.js
@@ -1,0 +1,1 @@
+module.exports = require('immutability-helper');

--- a/package.json
+++ b/package.json
@@ -79,6 +79,7 @@
     "preact": ">=5"
   },
   "dependencies": {
+    "immutability-helper": "^2.1.2",
     "preact-render-to-string": "^3.6.0",
     "preact-transition-group": "^1.1.0",
     "proptypes": "^0.14.3",

--- a/test/lib/update.js
+++ b/test/lib/update.js
@@ -1,0 +1,13 @@
+import update from '../../lib/update';
+
+describe('update', () => {
+	it('should export an function', () => {
+		expect(update).to.be.a('function');
+	});
+
+	it('should update arrays', () => {
+		const initialArray = [1, 2, 3];
+		const newArray = update(initialArray, {$push: [4]});
+		expect(newArray).to.eql([1, 2, 3, 4]);
+	});
+});

